collect/metric_objects.go

```diff
@@ -0,0 +1,50 @@
+package collect
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/numaproj-labs/numaflow-perfman/util"
+)
+
+const (
+	minutes = int(util.RateInterval / time.Minute)
+	seconds = int((util.RateInterval % time.Minute) / time.Second)
+)
+
+type MetricObject struct {
+	Query    string
+	Filename string
+	XAxis    string
+	YAxis    string
+}
+
+// Data forward metrics
+var InboundMessages = MetricObject{
+	Query:    fmt.Sprintf(`rate(forwarder_read_total{pipeline="perfman-base-pipeline"}[%dm%ds])`, minutes, seconds),
+	Filename: "inbound-messages",
+	XAxis:    "Unix Timestamp",
+	YAxis:    "Number of Messages Per Second",
+}
+
+// Latency metrics
+var ForwarderE2EP90 = MetricObject{
+	Query:    fmt.Sprintf(`histogram_quantile(0.9, rate(forwarder_forward_chunk_processing_time_bucket{pipeline="perfman-base-pipeline"}[%dm%ds])) / 1000000`, minutes, seconds),
```

Comment:
In the future when we support customized pipelines, we will need to dynamically configure the string. Please add a TODO here.